### PR TITLE
Web portal: Refine document

### PR DESCRIPTION
**Pull request by Gerhut**

### CODE REVIEW

File: webportal/README.md [hunk 1]
```diff
@@ -1,15 +1,89 @@
...
+
+N/A
+
+## High Availability
+
+Web portal is a stateless service, so it could be extended for high availability without any extra operation.
+
+## Runtime Requirements
+
+To run web portal on system, a [Node.js](https://nodejs.org/) 6+ runtime is required, with [npm](https://www.npmjs.com/) and [yarn](https://yarnpkg.com/)(JavaScript package manager) installed.
```
Comment (hwuu) [inline webportal/README.md]: I think 'runtime requirements' means the minimum/average/maximum CPU/memory consumption.

File: webportal/README.md [hunk 2]
```diff
@@ -1,15 +1,89 @@
...
+
+* [REST Server](../rest-server)
+* [Prometheus](../prometheus)
+* [Grafana](../grafana)
+* YARN
+* Kubernetes
+
+## Build
+
+*In current design, there is nothing to do in build stage.*
```
Comment (YanjieGao) [inline webportal/README.md]: show this service image build and push is better
https://github.com/Microsoft/pai/tree/dff283ae0292141790d87a4c42d75d62cfee81da/pai-management/src/webportal

Comment (Gerhut) [reply webportal/README.md]: @YanjieGao Should I assume that readers of this readme will deploy standalone or by pai-management, or both? In fact I'm not clear about this.

Comment (YanjieGao) [reply webportal/README.md]: @sterowang , bin, qixiang's question need your help

**Pull request merged**